documentation/src/main/asciidoc/ch11.asciidoc

```diff
@@ -694,17 +694,25 @@ mechanism to register additional constraint definitions. All you have to do is t
 _javax.validation.ConstraintValidator_ to _META-INF/services_. In this service file you list the
 fully qualified classnames of your constraint validator classes (one per line). Hibernate Validator
 will automatically infer the constraint types they apply to.
-See <<example-using-service-file-for-constraint-definitions>> for an example.
+See <<example-using-service-file-for-constraint-definitions,Constraint definition via service file>>
+for an example.
 
 [[example-using-service-file-for-constraint-definitions]]
-.Using _META-INF/services/javax.validation.ConstraintValidator_ for constraint definitions
+._META-INF/services/javax.validation.ConstraintValidator_
+
 ====
 [source]
 ----
+# Assuming a custom constraint annotation @org.mycompany.CheckCase
 org.mycompany.CheckCaseValidator
 ----
 ====
 
+To contribute default messages for your custom constraints, place a _ValidationMessages.properties_
```

Comment:
> It'd be a defined bundle name. I guess you could consider that as a contract.

Ahh, got you now. You just want to use a different name? What would that be? 

> Actually it's re-using the defined name from the spec with slightly other semantics I'm a bit concerned about

Where is your concern about this?